devsetup/Makefile

```diff
@@ -83,6 +84,41 @@ crc_cleanup: ## Destroys the CRC env, but does NOT clear ( --clear-cache ) the c
 	sudo ${CLEANUP_DIR_CMD} /etc/pki/ca-trust/source/anchors/crc-router-ca.pem
 	sudo update-ca-trust
 
+.PHONY: add_nics_crc
+add_nics_crc: remove_nics_crc
+	# Add data interface
+	MAC_ADDRESS=$(shell echo -n 52:54:00; dd bs=1 count=3 if=/dev/random 2>/dev/null | hexdump -v -e '/1 "-%02X"' | tr '-' ':'); \
```

Comment:
how did you define the mac prefix? should we put it in a variable?

Reply:
The mac prefix is the same used by the target `crc_attach_default_interface`

Reply:
Yes. Which makes me think that maybe keeping it in a variable and then reusing it here. (And in other places) is a good idea.